Formula/matterhorn.rb

```diff
@@ -0,0 +1,20 @@
+class Matterhorn < Formula
+  desc "Unix terminal client for the Mattermost chat system"
+  homepage "https://github.com/matterhorn-chat/matterhorn"
+  url "https://github.com/matterhorn-chat/matterhorn.git", :tag => "50200.1.1"
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
+
+  def install
+    system "rm", "cabal.project"
+    system "cabal", "new-update"
+    system "cabal", "install", "tasty-quickcheck"
+    system "./build.sh"
+    bin.install "dist-newstyle/build/x86_64-osx/ghc-8.4.4/matterhorn-50200.1.1/x/matterhorn/build/matterhorn/matterhorn"
```

Comment:
That path will break at every update 😢

Reply:
I'll check with the developers if they can simplify this.